openedx/core/djangoapps/programs/tests/test_models.py

```diff
@@ -70,3 +71,51 @@ def test_is_student_dashboard_enabled(self, _mock_cache):
 
         self.create_config(enabled=True, enable_student_dashboard=True)
         self.assertTrue(ProgramsApiConfig.current().is_student_dashboard_enabled)
+
+    @ddt.data(
+        (True, True, 0),
+        (False, True, 0),
+        (False, True, 1),
+        (True, False, 0),
+    )
+    @ddt.unpack
+    def test_is_cache_enabled_returns_false(self, enabled, is_dashboard, ttl, _mock_cache):
+        """Verify that the method 'is_cache_enabled' returns false if
+        'cache_ttl' value is 0 or config is not enabled.
+        """
+        self.assertFalse(ProgramsApiConfig.current().is_cache_enabled)
+
+        self.create_config(
+            enabled=enabled,
+            enable_student_dashboard=is_dashboard,
```

Comment:
This would be easier to read if the variable name matched the name of the model field, `enable_student_dashboard`.